src/main/java/org/myrobotlab/service/InMoov2Head.java

```diff
@@ -502,15 +472,12 @@ public void test() {
       eyelidRight.moveToBlocking(1.0);
   }
 
-  public void autoBlink(boolean param) {
-    if (blinkEyesTimer != null) {
-      blinkEyesTimer.cancel();
-      blinkEyesTimer = null;
-    }
-    if (param) {
-      blinkEyesTimer = new Timer();
-      new blinkEyesTimertask().run();
-    }
+  /**
+   * FIXME - implement
+   * @param b
+   */
+  public void autoBlink(boolean b) {

```

Comment:
no functionality anymore..

Reply:
yeah gutted from within, for good reason - but should be re-implemented with sane implementation and config !